src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -24,4 +35,30 @@ public List<EventDTO> updateEvent() {
     public List<EventDTO> getEvents() {
         return null;
     }
-}
+
+    @Override
+    public List<DetailsParticipantDTO> getRegisteredParticipants(Long eventId) {
+        Optional<Event> eventOptional = eventRepository.findById(eventId);
+        List<DetailsParticipantDTO> detailsParticipantDTOList = new ArrayList<>();
+
+        if (eventOptional.isPresent()) {
+            Event event = eventOptional.get();
+            List<Participant> participants = event.getParticipants();
+
+            if (participants == null) {
+                return detailsParticipantDTOList;
+            }
+
+            for (Participant participant : participants) {
+                DetailsParticipantDTO detailsParticipantDTO = new DetailsParticipantDTO();
```

Comment:
Can we maybe use a library like `ObjectMapper` to help us with entity to DTO conversion?